src/extension.ts

```
@@ -218,8 +230,76 @@ function migrateOldSettings(): void {
     }
   });
 }
+const BLOOP_DISCONNECT_TIMEOUT_MS = 5000;
+
+/**
+ * Register this window as an instance that has Metals (and possibly Bloop) running.
+ * Call once when the language client is set (machine-scoped: one Bloop per user).
+ */
+function registerBloopInstance(): void {
+  if (!bloopInstancesDir || currentBloopInstanceId) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)